at-dungnguyen/Final/model/leave.rb

```diff
@@ -0,0 +1,54 @@
+require 'pry'
+require 'mysql2'
+class Leave
+  attr_reader :id
+  attr_accessor :id_user, :reason, :total_date_off, :status, :publish_date
+  @@client = Mysql2::Client.new(host: '127.0.0.1', username: 'root', password: 'win112344', database: 'DungNguyen_HumanResources')
+
+  def initialize(id_user, reason, total_date_off, status, publish_date)
+    @id_user = id_user
+    @reason = reason
+    @total_date_off = total_date_off
+    @status = status
+    @publish_date = publish_date
+  end
+
+  def self.add_request(leave)
+    sql = "INSERT INTO `DungNguyen_HumanResources`.leave(id_user, reason, total_date_off, status, publish_date) VALUES(#{leave.id_user}, '#{leave.reason}', #{leave.total_date_off}, 0, '#{leave.publish_date}')"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/3c0bf6c6d3420a006a4c5cda03d7111306060898/at-dungnguyen/Final/model/leave.rb#L17"></span>Metrics/LineLength: Line is too long. [209/160] (https://github.com/rubocop-hq/ruby-style-guide#80-character-limits)</td>
    </tr>
  </tbody>
</table>